### ISSUE TITLE
Brave Shields - Trackers & Ads to Aggresive and Fingerprint to Strict mode - the option doesn't persist after closing the browser.

### ISSUE THREAD

**Issue by giwrgosmant** [brave/brave-browser#17411]
## Description 

In settings of Shields, If I change Trackers & ads blocked to Aggressive mode and Fingerprint to Strict mode and then clear all data/cookies/history and close the browser, it automatic goes back to Standard mode. Both in Android & Desktop.

## Steps to Reproduce

   1. Change the setting of Trackers & ads blocked to Aggressive mode and Fingerprint to Strict mode.
   2. Then clear all data/cookies/history and close the browser.
   3. Open the browser, and you will see the settings went back to Standard.

## Actual result:

It's going back to Standard mode.

## Expected result:
It should stay to Aggressive mode,

## Reproduces how often: 

Always, easy to reproduced,

## Desktop Brave version:

<html>
<body>

Brave | 1.27.111 Chromium: 92.0.4515.131 (Επίσημη έκδοση) (64-bit)
-- | --
Αναθεώρηση | 6b8d6c56ce21e38a72f7c4becb5abc1fa5134f29-refs/branch-heads/4515@{#1933}
OS | Windows 10 OS Version 2009 (Build 19043.1151)
JavaScript | V8 9.2.230.22
Παράγοντας χρήστη | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36
Γραμμή εντολών | "C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe" --enable-dom-distiller --disable-domain-reliability --no-pings --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --enable-features=LegacyTLSEnforced,WebUIDarkMode,AutoupgradeMixedContent,SafetyTip,WinrtGeolocationImplementation,PrefetchPrivacyChanges,PasswordImport,ReducedReferrerGranularity --disable-features=FledgeInterestGroups,HandwritingRecognitionWebPlatformApi,InterestCohortFeaturePolicy,FlocIdComputedEventLogging,HandwritingRecognitionWebPlatformApiFinch,EnablePasswordsAccountStorage,AutofillEnableAccountWalletStorage,FledgeInterestGroupAPI,EnableProfilePickerOnStartup,TextFragmentAnchor,IdleDetection,InterestCohortAPIOriginTrial,FederatedLearningOfCohorts,PrivacySandboxSettings,FirstPartySets,SubresourceWebBundles,NetworkTimeServiceQuerying,LangClientHintHeader,NotificationTriggers,WebOTP,LiveCaption,AutofillServerCommunication,SignedExchangePrefetchCacheForNavigations,SignedExchangeSubresourcePrefetch,DirectSockets,TrustTokens --flag-switches-begin --flag-switches-end
Διαδρομή εκτελέσιμου | C:\Program Files\BraveSoftware\Brave-Browser\Application\brave.exe
Διαδρομή προφίλ | C:\Users\User1\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default
Παραλλαγές | AdRewardsStudy:NextPaymentDayEphemeralStorageStudy:EnabledNativeCosmeticFilteringStudy:EnabledPermissionLifetimeReleaseStudy:Enabled

</body>
</html>

## Android Device details:
   - Install type (ARM, x86): ARM64
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: 11

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by Tonev** [comment]
Clearing `Site and Shields Settings` browsing data resets global `Brave Shields` settings.

Reported here: https://github.com/brave/brave-browser/issues/15347

**Comment by giwrgosmant** [comment]
> Clearing `Site and Shields Settings` browsing data resets global `Brave Shields` settings.
> 
> Regarding Android - you have to manually clear `Site settings`, the `Clear data on exit` feature doesn't reset the `Brave Shields` settings anymore.
> 
> Reported here: https://github.com/brave/brave-browser/issues/15347

You are right "Clear data on exit" does not reset Brave Shields back to standard mode.
However I want to manually clear data and since it's global settings, it should keep my settings like I set it. I hope it's gonna be fixed soon.

**Comment by KissesRefined** [comment]
+1
Manually clearing data does reset Brave Shields back to standard mode. Would love to see this fixed.

**Comment by superlex** [comment]
Any news?

**Comment by superlex** [comment]
@giwrgosmant 
Which options do you select when you manually clear data?

**Comment by superlex** [comment]
Try in this way:

![Screenshot_20231108_123226_Brave.png](https://github.com/brave/brave-browser/assets/1375223/0585397a-9135-447d-abd0-674c67dd8866)

**Comment by superlex** [comment]
> Indeed, at least in Android which I still use Brave, clearing without selecting "Site settings", strict modes persist.
> So the issue is with that particular option. Good to know, thx. I hope team would fix it soon.

You're welcome 😊
However, in Brave for Desktop this option is called something like "Site and Shield settings", so it's possible that this isn't a issue, but an expected behaviour. I don't know 🤷🏻‍♂️

**Comment by giwrgosmant** [comment]
The issue seems fixed.
Took a while...but thanks Brave team.
